lib/reverse_sentence.rb

```diff
@@ -1,6 +1,46 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
+# Time complexity: O(n)
+# Space complexity: O(1)
```

Comment:
Let me just say, this is an awesome solution!  You have the ideal time/space complexity.  Well done.  I only have one comment, check below.